README.md

```diff
@@ -75,7 +75,37 @@ This command is used to setup the main command (/notify) by adding in the a user
 
 # Running The Bot
 
+This section outlines the steps required to get this project up and running on your local dev environment.
+
+## Prerequisites
+
+1. [Install Docker](https://docs.docker.com/get-docker/)
+2. [Install Docker Compose](https://docs.docker.com/compose/install/)
+
 ## Environment Variables
 
+For your local development environment setup, you will require a `.env` file. Create this file in the root directory of the project.
+Copy the contents of the `.env.example` file into this file, and populate the value for the `GUB_TESTING_BOT_TOKEN` key.
+
+### Generating a GUB_TESTING_BOT_TOKEN
+
+We will need an API token to access the [BotsAPI from telegram](https://core.telegram.org/bots/api). In order to generate one:
+
+1. Open the Telegram app using the desktop or web version
+2. In the search bar, type in `BotFather` and select the verified profile of BotFather
+3. Start the bot and issue the command `/newbot` in the chat to create a new testing bot
+4. Give your testing bot a name. Note: the bot name must end in `bot` (for example: TetrisBot or tetris_bot)
+5. Copy the Bot API token from the response message and paste it into the `.env` file
+
+## Spinning up your bot
+
+Once you have generated the API token and have setup your `.env` file, run the following command from the root directory of the project:
+```
+docker-compose up --build
```

Comment:
Also one more thing Shreyas, can you add a bit more information about this command, for example:

"you only need to run `docker-compose up --build` when you first build the image for the project bot, but afterwards  you can simply run `docker-compose up` if you don't need to rebuild the project. To learn more about this, you can checkout the Dockerfile located in the root directory and also read more about docker-compose here (and then insert a link to a helpful resource)" 

I drafted that very quickly and very roughly so please reword that quite significantly